src/main/java/frc/robot/subsystems/AlgaeIntake.java

```diff
@@ -32,6 +31,7 @@ public class AlgaeIntake extends SubsystemBase {
 
   private Angle lastDesiredAngle = Degrees.zero();
 
+  MotionMagicVoltage motionRequest;
   PositionVoltage positionRequest;
```

Comment:
Right now motionRequest is null so the code will fail in sim if we try to run it.
```suggestion
  MotionMagicVoltage motionRequest = new MotionMagicVoltage(0);
```